packages/manager/apps/web/client/app/hosting/database/order/public/components/steps/db-categories-offers/controller.js

```diff
@@ -17,7 +17,14 @@ export default class WebHostingDatabaseOrderComponentsDbCategoriesOffersControll
   }
 
   getDbEngines() {
-    return this.model.dbCategory?.selectVersion?.engines || [];
+    return this.model.dbCategory?.selectVersion?.engines?.map((engine) => {
```

Comment:
no impact to return `undefined` instead of `[]` ?

Reply:
![image](https://github.com/ovh/manager/assets/146735326/8eec8210-4562-4993-ac58-90280aac2352)
Indeed it could be an issue

My proposal : 

![image](https://github.com/ovh/manager/assets/146735326/64ee15dd-4d1d-4390-85ac-080d18a14d7e)